be/src/exec/pipeline/olap_chunk_source.cpp

```diff
@@ -416,6 +416,8 @@ Status OlapChunkSource::_read_chunk_from_storage(RuntimeState* state, vectorized
         if (Status status = _prj_iter->get_next(chunk); !status.ok()) {
             return status;
         }
+        SCOPED_SET_CATCHED(true);
```

Comment:
better add to the `TRY_CATCH_ALLOC_SCOPE_START`

Reply:
fixed